.npmignore

```diff
@@ -1,16 +1,25 @@
 # standard ignore
+*config*
+*conf*
+
+apiExamples/
+demo/partials/
+.github/
 .gitignore
+.eslintrc
 .stylelintrc
-karma.conf.js
-rollup.config.js
-babel.config.js
-.github/
-index.html
+
+coverage/
+docs/
 scripts/
 test/
-demo/css
-demo/sass
-coverage/
+demo/**/*.css
+demo/**/*.scss
+demo/**/*.html
+demo/**/*.md
+
+index.html
 
 # ignore src
-src/**/*
+src/**/*.scss
+src/**/*.css
```

Comment:
Only these resources are to be updated at this time. All the other ones are still correct as this repo as not been totally updated.